samples/eShopLite/AppHost/appsettings.json

```diff
@@ -4,5 +4,10 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
+  },
+  "Secrets": {
+    "secretstore": {
+      "apikey": "fakeapikey"
+    }
```

Comment:
Shouldn't this come from user secrets in practice?

Reply:
Yep. That's what I'm thinking.